bin/iuf-skopeo-sync

```diff
@@ -82,10 +81,12 @@ process(){
             image_dir="$(dirname "$relative_manifest_path")"
             image_name="$(cut -d":" -f 1 <<< "$image_dir")"
             image_version="$(cut -d":" -f 2 <<< "$image_dir")"
-            echo "INFO recording results for ${image_name}:${image_version}"
+            echo "INFO Adding entry in 'records.yaml' file for image: ${image_name}:${image_version}"
             record_result "$image_name" "$image_version"
         done
     else
+        result=$(echo "$result" | sed -e 's/^/DEBUG /')
+        echo -e "DEBUG <skopeo sync --scoped --src dir --dest docker --dest-creds "${NEXUS_USERNAME:-admin}:${NEXUS_PASSWORD}" --dest-tls-verify=false /images/"$docker_dir" "${NEXUS_REGISTRY:-registry.local}"> failed with\n\n$result"        
```

Comment:
we should redact the `${NEXUS_PASSWORD}`

Reply:
Done